libs/providers/ofrep-web/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@openfeature/ofrep-web-provider",
+  "version": "0.0.1-alpha.0",
+  "dependencies": {
+    "tslib": "^2.3.0"
+  },
+  "main": "./src/index.js",
+  "typings": "./src/index.d.ts",
+  "scripts": {
+    "publish-if-not-exists": "cp $NPM_CONFIG_USERCONFIG .npmrc && if [ \"$(npm show $npm_package_name@$npm_package_version version)\" = \"$(npm run current-version -s)\" ]; then echo 'already published, skipping'; else npm publish --access public; fi",
+    "current-version": "echo $npm_package_version"
+  },
+  "peerDependencies": {
+    "@openfeature/web-sdk": ">=0.4.0"
```

Comment:
I think you need a peer dep on ofrep core?

Reply:
We can bundle it, but in that case we have to be aware if 2 modules bundle it, they might both export artifacts from them and then we will get the problems I described here.